AdministratorAdministration.yaml

```diff
@@ -938,12 +938,12 @@ paths:
                           type: string
                           description: >
                             'Own IP address
-                            from [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-tcp-s-bm-000/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-address/ipv-4-address]'
+                            from [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-tcp-s-000/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-address/ipv-4-address]'
                         oam-approval-port:
                           type: integer
                           description: >
                             'Own TCP port
-                            from [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-tcp-s-bm-000/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-address/local-port]'
+                            from [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-tcp-s-000/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-address/local-port]'
                       example:
```

Comment:
As per comment https://github.com/openBackhaul/ExecutionAndTraceLog/pull/153#discussion_r1047382078, can we also update TAR data in line 1148 (example of /v1/list-applications)

Reply:
Changed protocol to HTTP, and adapted IP address and port to match with release number 1.0.0.